### DISCUSSION TITLE
After disabling ldap users in Microsoft, users who have logged in to the client can still use it normally.

### DISCUSSION THREAD

**Discussion by Rocky-Jinxing** [rustdesk/rustdesk-server-pro#375] [General]
After disabling ldap users in Microsoft, users who have logged in to the client can still use it normally, no matter I log out of the client and reopen it, the account is still logged in. Only when I log out and re-enter the username and password to log in, I can't log in.

**Comment by Rocky-Jinxing** [comment]
I found that there was a heartbeat between the client and the rustdesk server user, but there was no heartbeat with the LDAP user.

**Reply by rustdesk** [reply]
Yes, current ldap are only used when login explicitly, it is not used if you auto login. This need to be improved, but low priority yet, we need listen to more users first.

**Reply by Rocky-Jinxing** [reply]
> Yes, current ldap are only used when login explicitly, it is not used if you auto login. This need to be improved, but low priority yet, we need listen to more users first.

This is a bug, which can result in our departing employees being able to use the account without logging out, posing a security risk.

**Reply by Rocky-Jinxing** [reply]
> > Yes, current ldap are only used when login explicitly, it is not used if you auto login. This need to be improved, but low priority yet, we need listen to more users first.
> 
> This is a bug, which can result in our departing employees being able to use the account without logging out, posing a security risk.

@rustdesk I hope you can fix this issue.
